### DISCUSSION TITLE
Upgrade pg_cron to 1.5.2

### DISCUSSION THREAD

**Discussion by mattaylor** [supabase/supabase#15937] [Feature Requests]
pg_cron provides support for sub minute schedules which I urgently need. Please this extension be upgraded.

**Comment by GaryAustin1** [comment]
Supabase tried to upgrade to it but ran into issues.
See the add and revert I posted here: https://github.com/orgs/supabase/discussions/15916

Also this is Supabase asking in pg_cron repository to resolve the issue:
https://github.com/citusdata/pg_cron/issues/274

I would watch the supabase/postgres github or the pg_cron issue to see if any progress happens...

**Comment by mattaylor** [comment]
thanks for the clarification. As the current limitations only appear to effect upgrading from 1.4, Is there any way you can support 1.5 for new installations of pg_cron without requiring existing 1.4 installations to be upgraded?
